### PR TITLE
Carl Yang team website draft

### PR DESCRIPTION
**Pull request by ericaychoi**

### CODE REVIEW

File: content/projects/2022-brain-nets/index.md [hunk 1]
```diff
@@ -4,6 +4,133 @@ date: 2022-06-27T11:36:49-04:00
...
 tags: ["Summer 2022"]
 ---
+# Overview: Comparing Shallow vs. Deep Brain Network Models
```
Comment (lruthotto) [inline content/projects/2022-brain-nets/index.md:10]: Suggestion: remove the 'overview'.

File: content/projects/2022-brain-nets/index.md [hunk 2]
```diff
@@ -4,6 +4,133 @@ date: 2022-06-27T11:36:49-04:00
...
+
+We are comparing and benchmarking the performance of graph kernels and graph neural networks applied to disease classification based on neuroimaging data.
```
Comment (lruthotto) [inline content/projects/2022-brain-nets/index.md:12]: Do you want to specify who is 'we'? I'll use the first paragraph later to link your paper and other materials you've created.

**Review by lruthotto — CHANGES_REQUESTED** [review]
Dear team

Thanks for sending your website draft. This all looks good, but I have a few suggestions. 
1) Keep in mind that this blog post will host a link to the paper. I could imagine that some things (like the tables and experimental setup) are best discussed there. That way, you can focus the website on a high level description of your REU experience and motivate people to read on. 
2) Please confirm that all the graphics were generated by you, so I know we have the copyright.

Thanks,
Lars

**Comment by ericaychoi** [comment]
Hi Professor Ruthotto,

Thank you for your feedback. We have some changes to the website that we
have made since the last pull request, so would it be okay for me to
incorporate your feedback and make another pull request later today?

Best,
Erica

**Comment by ericaychoi** [comment]
Hi Professor Ruthotto, 

My team has made some updates and have addressed your last feedback. Please let us know if there are any other concerns regarding our website. Thank you!

Best,
Erica

**Review by lruthotto — APPROVED** [review]
Thanks for the revision. This looks good to go! I may do some minor edits before uploading it.

**Pull request merged**